eng/versioning/version_client.txt

```diff
@@ -60,6 +60,8 @@ com.microsoft.azure:azure-spring-boot-metrics-starter;2.2.4;2.2.5-beta.1
 # Format;
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
+unreleased_com.azure:azure-core;1.6.0-beta.1
+unreleased_com.azure:azure-core-http-netty;1.6.0-beta.1
```

Comment:
this will need to be beta_ dependencies I think.
To ensure we pass build from source builds.

Reply:
As discussed in https://github.com/Azure/azure-sdk-for-java/pull/11780/files/2bfaa4971cde9c3482877aa3d96321f715a70d84..264b0fbd92bbdd25907b5789bfcbfa93b339616c#diff-b1a225c9ee81df78dc378973728e3ca6, the beta_ tags will be applied once azure-core:1.6.0-beta1 is released. 
Also, as the versioning stuff here are only applied to azure-core's dependants, yet the only library being shipped out of this branch is azure-core so this shouldn't make a difference.